### PR TITLE
Newsletter #120

### PR DESCRIPTION
**Pull request by geshan**

### CODE REVIEW

File: source/_posts/2016-01-07-newsletter-number-120.markdown [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+title: "Newsletter #120"
+date: 2016-01-07 15:26
+author: Geshan Manandhar
+comments: true
+categories: [TechThursday, symfony, angular2, cli, composer, css, docker, documentation, http2, iesupport, javascript, performance, php, programming, react, reactjs, symfony, tdd, testing, webcomponents, webpack]
+---
+
+>
+> Hello everyone, 
+> for the first newsletter of 2016, we have the final results from our ping pong (TT) competition.
```
Comment (cirpo) [inline source/_posts/2016-01-07-newsletter-number-120.markdown]: from or of?

Comment (razanbilwani) [inline source/_posts/2016-01-07-newsletter-number-120.markdown]: *For

File: source/_posts/2016-01-07-newsletter-number-120.markdown [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+{% img center /images/posts/ping-pong-final.jpg %}
+
+> This is how the tournament unfolded in the past weeks:
+
+<!-- more -->
+
+{% img center /images/posts/ping-pong-tie-sheet.jpg %}
+
+> And yes here is your weekly dose of links we came across in the first week of 2016! Enjoy!!
```
Comment (razanbilwani) [inline source/_posts/2016-01-07-newsletter-number-120.markdown]: And, yes, here...

File: source/_posts/2016-01-07-newsletter-number-120.markdown [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+* [welcome to future of web application delivery — medium](https://medium.com/@ryanflorence/welcome-to-future-of-web-application-delivery-9750b7564d9f)
+
+* [what good code is? – carlos alexandro becker](http://carlosbecker.com/posts/good-code/)
+
+* [why did i write a machine learning book in ruby?](http://matthewkirk.com/why-did-i-write-a-machine-learning-book-in-ruby/?utm_content=buffer4f976)
+
+* [writing clear code, not clever code](https://medium.com/@mikesherov/writing-clear-code-not-clever-code-d6b90353a3c5)
+
+* [~2015 in review — sebastian mckenzie](https://medium.com/@sebmck/2015-in-review-51ac7035e272)
```
Comment (shidhincr) [inline source/_posts/2016-01-07-newsletter-number-120.markdown]: Repeated link

File: source/_posts/2016-01-07-newsletter-number-120.markdown [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+
+* [python moving to github](https://mail.python.org/pipermail/core-workflow/2016-January/000345.html?utm_content=buffer11c43)
+
+* [react router’s future — rackt and roll — medium](https://medium.com/rackt-and-roll/react-router-s-future-c026c0f2874f)
+
+* [react’s jsx: the other side of the coin — medium](https://medium.com/@housecor/react-s-jsx-the-other-side-of-the-coin-2ace7ab62b98)
+
+* [recipe-oriented documentation — medium](https://medium.com/@andrewingram/recipe-oriented-documentation-1601caed8a44)
+
+* [recipe-oriented documentation](https://medium.com/@andrewingram/recipe-oriented-documentation-1601caed8a44)
```
Comment (shidhincr) [inline source/_posts/2016-01-07-newsletter-number-120.markdown]: Repeated link

**Comment by geshan** [comment]
removed repeated links. merging.

**Pull request merged**